doc/tex/secciones/03_analisis.tex

```diff
@@ -243,106 +243,18 @@ \subsection{HU-06 - Criterios de evaluación}
     \end{adjustbox}
     \caption{HU-06. Criterios de evaluación.}
 \end{table}
-  
-\section{User journeys}
-
-En esta sección se enumeran los \textit{user journeys} tanto para el uso del sistema, como para la ampliación del repertorio de comandos.
-
-\subsection{Uso del sistema}
 
-\subsubsection{Crear un bot}
 
-\begin{enumerate}
-	\item El usuario crea una aplicación de \textit{Discord} en el \href{https://discord.com/developers/applications}{portal de desarrolladores} y obtiene un \textit{token} para un bot.
-	\item El usuario provee al sistema de este \textit{token} y de un nombre.
-	\item[!] Si el nombre o el \textit{token} se encuentran en uso por otro bot, el sistema cancela la creación.
-	\item El bot es creado en el sistema, quedando disponible para ser configurado con comandos o para ser desplegado.
-\end{enumerate}
-
-\subsubsection{Modificar un bot}
+\section{User journeys}
 
-Se puede modificar un bot de dos maneras:
+En esta sección se enumeran algunos \textit{user journeys} que podría realizar el usuario programador.
 
 \begin{itemize}
-	\item Parámetros del bot.
-	\begin{enumerate}
-		\item El usuario provee al sistema de datos del bot a actualizar.
-		\item[!] Si los datos coinciden con los actuales o con los de otro bot, el sistema cancela la modificación.
-		\item[!] Si el bot se encuentra desplegado, el sistema cancela el despliegue previa modificación.
-		\item El sistema modifica los detalles del bot.
-	\end{enumerate}
-	
-	\item Comandos del bot.
-	\begin{enumerate}
-		\item El usuario provee al sistema de los identificadores de los comandos personalizados que quiere agregar o eliminar del bot.
-		\item[!] Si el bot se encuentra desplegado, el sistema cancela el despliegue previa modificación.
-		\item El sistema modifica los comandos del bot.
-	\end{enumerate}
+	\item El programador, cuando se encuentra interactuando con otros usuarios en Discord, detecta una serie de funcionalidades (como enviar mensajes de alerta a todos los usuarios, o asignar un rol a cierto usuario) que sería interesante agregar al servidor. Accede al sistema y configura estas tareas en un bot, el cual agrega al servidor de Discord.
+	\item El programador comprueba cada semana las necesidades que tienen los usuarios de los servidores de Discord donde ha agregado bots previamente. Si estas funcionalidades están obsoletas, las actualiza en el sistema; si son insuficientes agrega nuevas; y si ya no son necesarias las elimina.
```

Comment:
¿Por qué hace eso el administrador? ¿Cómo lo hace? ¿Cómo defines necesidades? ¿Las funcionalidades ocupan espacio? ¿Suponen un gasto?
Es que también me falta que expliques "serie de funcionalidades". Si no eres preciso, la audiencia no puede saber si esto lo estás haciendo y realmente estás añadiendo algún valor a un cliente de discord o qué es lo que estás haciendo.
En otro orden de cosas, lo que dices aquí no tiene mucho sentido, que sólo se mantengan las "funcionalidades" (sea como sea la definición de eso) que se han usado la última semana. Si sigues así te quedas sin funcionalidades.
Y también es un ejemplo de lo que se está haciendo desde el principio: tienes claro que quieres hacer, y tratas de ajustar las HUs, hitos, user journeys y demás de forma que lo que querías hacer desde el principio tenga sentido. No ha funcionado hasta ahora, y no va a funcionar.
Mira [este tutorial, por ejemplo, sobre todo los pasos 2 y 3](https://uxplanet.org/a-beginners-guide-to-user-journey-mapping-bd914f4c517c)
Recuerda que siempre un producto tiene que crear valor para el cliente o clientes, ofrecerle cierto beneficio. Si un user journey no muestra claramente cuál es el problema y ofrece contexto, no es útil y puedes simplemente suprimirlo.